### DISCUSSION TITLE
Unable to make the model - "no such file or directory"

### DISCUSSION THREAD

**Discussion by jamezpowell** [ggml-org/whisper.cpp#2622] [Q&A] [answered]
Hi, I am trying to install the ggml-large-v3-turbo model on my M2 Ultra mac studio, but I keep getting this error:

james@macstudio whisper.cpp % make -j large-v3-turbo
bash ./models/download-ggml-model.sh large-v3-turbo
Downloading ggml model large-v3-turbo from 'https://huggingface.co/ggerganov/whisper.cpp' ...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1202  100  1202    0     0   6193      0 --:--:-- --:--:-- --:--:--  6195
100 1549M  100 1549M    0     0  26.9M      0  0:00:57  0:00:57 --:--:-- 28.2M
Done! Model 'large-v3-turbo' saved in '/Users/james/whisper.cpp/models/ggml-large-v3-turbo.bin'
You can now use it like this:

  $ ./main -m /Users/james/whisper.cpp/models/ggml-large-v3-turbo.bin -f samples/jfk.wav

cmake -B build
make: cmake: No such file or directory
make: *** [large-v3-turbo] Error 1
james@macstudio whisper.cpp % 

james@macstudio whisper.cpp % make
cmake -B build
make: cmake: No such file or directory
make: *** [build] Error 1
james@macstudio whisper.cpp %

**Comment by misutoneko** [comment, marked as answer]
No CMake installed?
